pkg/stage.sh

```diff
@@ -45,9 +45,6 @@ cp -r opt build/opt
 cp -r etc build/etc
 cp -r usr build/usr
 
-echo "Stage the JRE..."
```

Comment:
update README.md to reflect that you no longer CP the JRE into the docker container